### ISSUE TITLE
Add an option to configure the request.

### ISSUE THREAD

**Issue by amanvishnani** [vasani-arpit/cordova-plugin-downloadmanager#1]
It would be great  if we could add headers to the request. I found a public method `addRequestHeader(String header, String value)` which can help us achieve it [here](https://developer.android.com/reference/android/app/DownloadManager.Request.html).

Can we add option to change the download path? If it is possible it would be great.

Thanks.

**Comment by vasani-arpit** [comment]
Yes, we should add headers. I don't think we can change the download path.
Will Update soon.

Thanks.
